src/views/CreateNew/DeployView.vue

```diff
@@ -74,12 +74,20 @@
                     </div>
                   </div>
                 </div>
-                <span
-                  class="text-sm font-normal text-color-secondary"
-                  v-if="isUnfinished"
-                >
-                  Project started {{ seconds }}s ago
-                </span>
+                <div v-if="isUnfinished">                  
+                  <span
+                    class="text-sm font-normal text-color-secondary"
+                    v-if="!isGreaterThanMinute"
+                  >
+                    Project started {{ timer }}s ago
+                  </span>
+                  <span
+                    class="text-sm font-normal text-color-secondary"
+                    v-if="isGreaterThanMinute"
```

Comment:
sugestão: usar v-else, pois assim evitamos a dupla verificação.

```suggestion
                    v-else
```